nushell_completions.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013-2022 The Cobra Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cobra
+
+import (
+	"bytes"
+	"io"
+	"os"
+)
+
+func (c *Command) GenNushellCompletion(w io.Writer) error {
+	buf := new(bytes.Buffer)
+	WriteStringAndCheck(buf, `
+# An external configurator that works with any cobra based
+# command line application (e.g. kubectl, minikube)
+let cobra_configurator = {|spans| 
```

Comment:
Let's name this `cobra_completer` as you have put in the documentation

Reply:
Oops, good catch!